docs/book/src/clusteropenstack/configuration.md

```diff
@@ -129,6 +130,21 @@ In order to access cluster nodes via SSH, you must either
 [access nodes through the bastion host](#accessing-nodes-through-the-bastion-host-via-ssh)
 or [configure custom security groups](#security-groups) with rules allowing ingress for port 22.
 
+## ORC
+
+ORC ([OpenStack Resource Controller](https://github.com/k-orc/openstack-resource-controller)) provides a set of Kubernetes controllers and is required by CAPO to 
+manage some OpenStack resources. ORC is a separate project and is not part of CAPO, therefore it needs to be installed separately.
+
+To install ORC, run the following command:
+
+```bash
+ORC_VERSION=v1.0.0
+kubectl apply -f https://github.com/k-orc/openstack-resource-controller/releases/download/$ORC_VERSION/install.yaml
+```
```

Comment:
I wonder if it's also worth mentioning the kustomize module we publish:
```bash
$ kubectl apply --server-side -k "https://github.com/k-orc/openstack-resource-controller/dist?ref=${ORC_VERSION}"
```

This would be in addition to what you wrote above. @mandre what do you think?

Reply:
I've added it.

Reply:
Looks good. I wonder if we should document an installation method where we change the default (for instance log level). It should probably go into the ORC documentation directly rather than a consumer like CAPO though.